webui/react/src/routes/routes.ts

```diff
@@ -115,6 +115,12 @@ const routes: RouteConfig[] = [
     path: '/reload',
     title: 'Reload',
   },
+  {
```

Comment:
we might wanna go with a more specialized route such as `job-queue` to leave jobs unused. I could see us having another view for jobs

Reply:
100% agree, `job-queue` sounds like good alternative